.github/workflows/notify-release.yml

```diff
@@ -0,0 +1,26 @@
+name: PRs not yet released checker
```

Comment:
Might recommend a shorter name for GitHub actions browsing.

`Unreleased PRs' or 'Unreleased PR Check'?